doc/modules/preprocessing.rst

```diff
@@ -565,7 +563,7 @@ example, these intervals are defined as:
  Based on these bin intervals, ``X`` is transformed as follows::
 
   >>> est.transform(X)                      # doctest: +SKIP
-  array([[ 0., 2., 1.],
+  array([[ 0., 1., 1.],
```

Comment:
If I understand correctly, the result change because we're now using ``strategy='quantile'``, right?
If so, I think we need to modify the intervals above, otherwise they're not consistent:
```
  - feature 1: :math:`{[-\infty, 0), [0, 3), [3, \infty)}`
  - feature 2: :math:`{[-\infty, 4), [4, 5), [5, \infty)}`
  - feature 3: :math:`{[-\infty, 13), [13, \infty)}`
```

Reply:
Good catch !